tests/E2E Tests/WebAppUiTests/UiTestNoParallelization.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Xunit;
+
+namespace WebAppUiTests
+{
+    [CollectionDefinition(nameof(UiTestNoParallelization), DisableParallelization = true)]
+    public class UiTestNoParallelization
```

Comment:
Is it possible to use just a string ? Seems like all usages are `nameof(UiTestNoParallelization)` which is just "UiTestNoParallelization"

Reply:
It can, I did it this way to make a clearer route for extensibility in the future as we could add collection level fixtures rather than class level. Would it be better to make it just a string?

Reply:
I appreciate the feedback but going to leave it as is.